chain-storage-sqlite/src/lib.rs

```diff
@@ -106,13 +109,14 @@ where
     }
 }
 
+#[derive(Clone)]
```

Comment:
This will require `B` to be `Clone too. is it what we want?

Reply:
I think we won't need cloning at all once #239 is integrated.